src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -927,7 +927,6 @@ class CreateVmModal extends React.Component {
                 userPassword: this.state.userPassword,
                 rootPassword: this.state.rootPassword,
                 userLogin: this.state.userLogin,
-                useCloudInit: this.state.sourceType === CLOUD_IMAGE,
```

Comment:
What happened to this? It looks like nothing actually looks at this property any more anywhere, was this just some forgotten cleanup? (again → separate commit)

Reply:
Moved to separate commit.